frontend/simple/js/events.js

```diff
@@ -128,3 +133,112 @@ export class MailboxContract extends Events.HashableMailbox {
     }
   })
 }
+
+const api = {
+  // TODO: add ability to unregister listeners
+  '/v1/identity/setAttribute': async function ({contractId, name, value}) {
+    let latestHash = await sbp('backend/v1/latestHash', {contractId})
+    let attribute = new Events.HashableIdentitySetAttribute({attribute: {name, value}}, latestHash)
+    await sbp('backend/v1/publishLogEntry', {contractId, entry: attribute})
```

Comment:
All these calls to `publishLogEntry` need to be removed, [as per original issue requirements](https://github.com/okTurtles/group-income-simple/issues/257#issuecomment-308263188):

> 5. The queue itself then goes through the actions, and it, **and only it**, calls `publishLogEntry`. No place else in the code should call `publishLogEntry` (as far as I can tell)

Remember, as that comment gave as an example, [these log entries are often created in chunks/groupings](https://github.com/vijayee/group-income-simple/blob/1818f7da5104a1eb29fd4a4f79e6ee3daeda6c42/frontend/simple/views/SignUp.vue#L89-L119)